static/src/javascripts/projects/common/modules/commercial/dfp/private/render-advert.js

```diff
@@ -149,6 +151,9 @@ define([
 
             function callSizeCallback() {
                 var size = slotRenderEvent.size.join(',');
+                if (size === '0,0') {
```

Comment:
Yeah, neither option is ideal. I thought maybe for the purposes of clarity we could alias `0,0` to `adSize.oldFluid`, which callbacks to `onFluidAd`. Not great